### DISCUSSION TITLE
Should we put the docs into project?

### DISCUSSION THREAD

**Discussion by caishunfeng** [apache/dolphinscheduler#6569] [Ideas]
Now the docs of dolphin is another alone project named dolphinscheduler-website,  which is forgettable to sync.
So I think we can put the dolphinscheduler-website into dolphinscheduler as docs instead of alone document project.
And it will be better to add the "edit this page"  button on page, which can push contributors to edit quickly.
What do you think?

**Comment by ruanwenjun** [comment]
There is a mail thread discuss this. 
https://lists.apache.org/api/atom.lua?mid=ra2ff3449cf517f6cad993a4fb6a3f787862508a37d64a84c64a9518e@%3Cdev.dolphinscheduler.apache.org%3E

**Comment by zhongjiajie** [comment]
Yes, I start a discuss in mailing-list, but finally we think maybe is better to separate them for now.
